### PR TITLE
New Case Study: Connecting an AWS VPC to Azure VNet via Resource Manager

### PR DESCRIPTION
**Pull request by sedouard**
New Devops case study about connecting Azure VNet to AWS VPC.

### CODE REVIEW

**Comment by felixrieseberg** [comment]
Your case study is a litte short... ;-)

**Comment by sedouard** [comment]
@felixrieseberg any idea why GH is only reporting 8 changed lines?

**Comment by felixrieseberg** [comment]
I'll blame encoding. I hope that it doesn't change anything in practice.

That said, could you remove the first title though? It's added automatically, so you'd end up with a double title.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+This architecture looks sort of like this:
+
+![Figure 1](http://az731655.vo.msecnd.net/content/awsazurediagram.png)
+
+We ended up creating [a reusable template](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn) to quickly create a Virtual Network gateway that can be deployed using the [azure cli](https://npmjs.org/azure-cli) from any kind of dev machine. You can find the [general template here](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn), and the AWS-[specific template here](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy.json). Both templates are essentially the same, with the primary difference being the variable naming.
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Not sure you need the reusable template link as you link to the templates shortly after. If you do keep it it reads like it would actually point to the one you made not the general template.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+It turns out, these Virtual Network (VNet) gateways can actually be connected to other clouds, in addition to on-premises VPNs. Today [Taplytics](https://taplytics.com) runs entirely on AWS and doesn't actually have an on-premises deployment. They would like to gradually bring some of their workload onto Azure as an extension of their existing AWS infrastructure.
+
+Using Azure Resource Manager's Virtual Network Gateways, we can actually connect an Azure Virtual Network to an AWS [Virtual Private Cloud](https://aws.amazon.com/vpc/), allowing traffic to flow from machines within the Azure VNet to the machines on the AWS VPC, and vice versa. By leveraging open source VPN software such as [StrongSwan](https://strongswan.org) we create an IPSec connection to the Azure gateway using a shared key. The strong swan instance within AWS not only can connect to Azure but can also be used to facilitate traffic to other nodes within the AWS VPC by [configuring forwarding](https://wiki.strongswan.org/projects/strongswan/wiki/ForwardingAndSplitTunneling) and using appropriate routing rules for the VPC.
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Is this only possible the the Resource Manager virtual network?

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+![Figure 5](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss4.png)
+
+Now, in your VPC, allocate a new Elastic IP (public ip) address and associate it with the instance:
+
+
+![Figure 6](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss5.png)
+![Figure 7](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss6.png)
+
+
+If you haven't already, its a good idea to change or at least add the hostname of your EC2 instance to it's `/etc/hosts` file (pointing to `127.0.0.1`) as StrongSwan's IPsec command seems to look for your current instance's hostname on the network:
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Grammar nit: I think you've got "it's" and "its" flipped in the study.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+Be sure to associate the routing table with your VPC and your Amazon VPC Subnet.
+
+Next, go to your security group associated with the ec2 instance and add 2 inbound custom UDP rules for ports 500 and 4500. The source for both should be a fixed single address CIDR block pointing your Azure VirtualNetworkGateway public IP formatted as: `[Azure VNet Public IP]/32`.
+
+![Figure 9](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss8.png)
+
+### Setting up Strongswan
+
+Finally, we need to setup the EC2 instance as a StrongSwan VPN server for the VPC. We can do this by `ssh`'ing into the instance and setting up Strongswan:
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Nit: Would be nice to be consistent with the strongswan casing. On their site they use "strongSwan"

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+
+Using the AWS console, create a new VPC, you can give a big address space such as `10.0.0.0/16`: 
+
+![Figure 2](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss1.png)
+
+Create a new subnet and associate it with the given vpc. Any subnet size will do as long as its within a subset of the ip address space defined in the previous step. I'm going with `10.0.0.0/24`.
+
+![Figure 3](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss2.png)
+
+Then launch an EC2 Ubuntu instance with the VPC and subnet created in the step above:
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: You might want to make this a separate section. Above you mention that one might skip the VPC setup section altogether if they have one. But this portion of the setup is about making the VPC (setup above or existing) externally accessible by creating the EC2 instance, giving it a public IP and disabling the source/destination checks.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+First, fill in the required parameters defined by [azuredeploy-parameters.json](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy-parameters.json). This includes `vpcGatewayIpAddress`, `vpcAddressPrefix`, `sharedKey` and `adminPassword`. There's also lots of other parameters you can change which you can find within the [template file](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy.json). The most important parameters are `vpcGatewayIpAddress` which is the public IP of the EC2 instance created above and `vpcAddressPrefix` which is the CIDR block of the VPC subnet.
+
+To deploy the parameters using the [azure cli](https://npmjs.com/azure-cli) create a new resource group then deploy the template to it.
+
+```
+azure group create aws2azure westus
+azure group deployment create aws2azure --template-file azuredeploy.json --parameters-file azuredeploy-parameters.json
+```
+
+Afterwards you can verify within the Resource Explorer blade on portal.azure.com that the resource group you deployed to contains a VirtualNetworkGatewayConnection to the AWS VPC.
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Make portal.azure.com a link. It could even go directly to the Resource Group list.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+Now, modify the strongswan configuration file `/etc/ipsec.conf` with a new connection called `azure`:
+
+```
+conn azure
+  authby=secret
+  type=tunnel
+  leftsendcert=never
+  left=10.0.0.28
+  leftsubnet=10.0.0.0/24
+  #leftnexthop=%defaultroute
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Comment can be removed to avoid noise in the instructions

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+  right=[AZURE GATEWAY PUBLIC IP]
+  rightsubnet=10.3.0.0/24
+  keyexchange=ikev2
+  ikelifetime=10800s
+  keylife=57m
+  keyingtries=1
+  rekeymargin=3m
+  #pfs=no
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Comment can be removed to avoid noise in the instructions

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+```
+ping [AWS Instance Local IP]
+64 bytes from 10.0.0.232: icmp_seq=892 ttl=62 time=26.1 ms
+64 bytes from 10.0.0.232: icmp_seq=893 ttl=62 time=26.2 ms
+64 bytes from 10.0.0.232: icmp_seq=894 ttl=62 time=25.8 ms
+...
+64 bytes from 10.0.0.232: icmp_seq=896 ttl=62 time=26.1 ms
+```
+
+You now have two subnets that are connected, one on AWS the other on Azure. You can use this connection for any type of communication that needs to be done between the two deployments, but be sure to modify your settings to allow traffic from other needed ports.
```
Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Consider splitting this off into a section; It could be a summary one. It feels like this is part of the testing section and makes me feel a bit like the case study ends abruptly.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 5 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: I feel like this is important. I want the case study to be very actionable to the developer, showing what you can do **now**. Explaining how to setup a VPC, although trivial for savvy AWS users may not be for someone new to IaaS in general. The entire case study is about connecting a VPC to Azure VNet so I feel that explaining how to create the VPC is worth while. I may just remove the 'you can skip this' sentence, since someone who already knows how to do this will skip it anyhow.

Comment (obsoleted) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: I agree. To be clear I wasn't suggesting removing anything. Just that if somebody does skip the section they may end up skipping too much if they have only setup a VPC and not setup a VM with external address, etc.. Removing the skipping note would work also.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 9 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Thanks! Added a *Next Steps* section.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 8 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Done!

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 6 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Done!

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 4 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Great eye!

**Comment by sedouard** [comment]
@obsoleted address all your feedback. Thanks!

**Comment by obsoleted** [comment]
Nice! :+1:   LGTM

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+
+Using Azure Resource Manager's Virtual Network Gateways, we can actually connect an Azure Virtual Network to an AWS [Virtual Private Cloud](https://aws.amazon.com/vpc/), allowing traffic to flow from machines within the Azure VNet to the machines on the AWS VPC, and vice versa. By leveraging open source VPN software such as [strongSwan](https://strongswan.org) we create an IPSec connection to the Azure gateway using a shared key. The strong swan instance within AWS not only can connect to Azure but can also be used to facilitate traffic to other nodes within the AWS VPC by [configuring forwarding](https://wiki.strongswan.org/projects/strongswan/wiki/ForwardingAndSplitTunneling) and using appropriate routing rules for the VPC.
+
+This architecture looks sort of like this:
+
+![Figure 1](http://az731655.vo.msecnd.net/content/awsazurediagram.png)
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: who owns this domain? should we host the images ourselves?

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+
+We ended up creating [a reusable template](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn) to quickly create a Virtual Network gateway that can be deployed using the [azure cli](https://npmjs.org/azure-cli) from any kind of dev machine. You can find the [general template here](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn), and the AWS-[specific template here](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy.json). Both templates are essentially the same, with the primary difference being the variable naming.
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Consider mentioning that the template is for ARM. 

> [a reusable template](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn) to quickly create a Virtual ....You can find the [general template here](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn)

The two links reference the same content. Was a bit confusing at first as I thought the "reusable" template was different than the "general" template. Maybe get rid of the first link?

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+
+## How to Connect AWS VPC to Azure Virtual Network
+
+### Setting up your VPC Part I
+
+Using the AWS console, create a new VPC, you can give a big address space such as `10.0.0.0/16`: 
+
+![Figure 2](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss1.png)
+
+Create a new subnet and associate it with the given vpc. Any subnet size will do as long as its within a subset of the ip address space defined in the previous step. I'm going with `10.0.0.0/24`.
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Nit.
 > Create a new subnet and associate it with the given *vpc*.

Capitalized VPC?

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 11 above)
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Suggest clarifying that the ARM template can be deployed using, in addition to the CLI that you mentioned, through the portal, powershell, and REST API (https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-deploy/)

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+vim /etc/hostname
+vim /etc/hosts
+sudo service hostname restart
+```
+
+### Deploying the Azure VNet & Gateway
+
+Deploying the Azure VNet and connection is made easy with the included resource manager template. This template includes a VirutalNetworkGateway, a LocalNetworkGateway and a VirtualNetworkGatewayConnection, as well as an instance to connect to an existing AWS VPC.
+
+First, fill in the required parameters defined by [azuredeploy-parameters.json](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy-parameters.json). This includes `vpcGatewayIpAddress`, `vpcAddressPrefix`, `sharedKey` and `adminPassword`. There's also lots of other parameters you can change which you can find within the [template file](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy.json). The most important parameters are `vpcGatewayIpAddress` which is the public IP of the EC2 instance created above and `vpcAddressPrefix` which is the CIDR block of the VPC subnet.
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Consider combining the explanation of the parameters when you first introduce them.

>First, fill in the required parameters defined by [azuredeploy-parameters.json](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy-parameters.json). This includes:
*  `vpcGatewayIpAddress` - public IP of the EC2 instance created above
* `vpcAddressPrefix` - CIDR block of the VPC subnet

> There are other parameters you can change which you can find in the [template file](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy.json)

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+
+![Figure 1](http://az731655.vo.msecnd.net/content/awsazurediagram.png)
+
+We ended up creating [a reusable template](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn) to quickly create a Virtual Network gateway that can be deployed using the [azure cli](https://npmjs.org/azure-cli) from any kind of dev machine. You can find the [general template here](https://github.com/Azure/azure-quickstart-templates/tree/master/201-site-to-site-vpn), and the AWS-[specific template here](https://github.com/sedouard/aws-vpc-to-azure-vnet/blob/master/azuredeploy.json). Both templates are essentially the same, with the primary difference being the variable naming.
+
+The great part is that although this works great for Taplytics' scenario, it also works well for anyone wanting to connect their existing cloud deployment to any Azure service that can be placed onto an Azure VNet. You can take the template, modify, change or reuse it to fit your specific needs.
+
+## How to Connect AWS VPC to Azure Virtual Network
+
+### Setting up your VPC Part I
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Nit: "Setting up your VPC (Part I): **Deploying**" then Part II: **Configuration**?

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 15]
```diff
@@ -0,0 +1,196 @@
...
+
+We need some details from the Azure deployment in order to complete configuring the VPC on the AWS side. First, we need to add a route to the VPC routing table and associate it with our subnet. 
+
+Add a route to the route table associated with the VPC, with the Azure Subnet CIDR as the destination, and the EC2 instance as the target:
+
+![Figure 8](http://az731655.vo.msecnd.net/content/azure-vnet-aws-vpc/ss7.png)
+
+Be sure to associate the routing table with your VPC and your Amazon VPC Subnet.
+
+Next, go to your security group associated with the ec2 instance and add 2 inbound custom UDP rules for ports 500 and 4500. The source for both should be a fixed single address CIDR block pointing your Azure VirtualNetworkGateway public IP formatted as: `[Azure VNet Public IP]/32`.
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Consider elaborating on the significance of these ports.

ie. default ports used by strongSwan for IKE and NAT-T

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 16]
```diff
@@ -0,0 +1,196 @@
...
+ipsec status
+Security Associations (1 up, 0 connecting):
+       azure[1]: ESTABLISHED 3 seconds ago, 10.0.0.28[10.0.0.28]...AZURE PUBLIC IP[AZURE PUBLIC IP]
+       azure{1}:  INSTALLED, TUNNEL, ESP in UDP SPIs: c3e607f3_i 280cade0_o
+       azure{1}:   10.0.0.0/24 === 10.3.0.0/24 
+```
+
+### Testing the Connection
+
+We've already created an Ubuntu instance in Azure by deploying the template. Create another Ubuntu instance on AWS, associated with the same VPC, subnet and security group.
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Suggest rewording:

> We've already created an Ubuntu instance in Azure by deploying the template.

Maybe... The Azure Resource template deployed earlier also provisions an Ubuntu VM that can be used to test the connection.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+
+You should see something similar when you ping the the AWS instance from Azure:
+
+```
+ping [AWS Instance Local IP]
+64 bytes from 10.0.0.232: icmp_seq=892 ttl=62 time=26.1 ms
+64 bytes from 10.0.0.232: icmp_seq=893 ttl=62 time=26.2 ms
+64 bytes from 10.0.0.232: icmp_seq=894 ttl=62 time=25.8 ms
+...
+64 bytes from 10.0.0.232: icmp_seq=896 ttl=62 time=26.1 ms
```
Comment (jpoon) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Are the two ping responses supposed to have the same IPs? :)

**Comment by jpoon** [comment]
Aside from the comments I mentioned, LGTM.

**Comment by felixrieseberg** [comment]
@sedouard: Let me know when you consider this ready to merge!

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 10 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: @jpoon lololol! That sketchyu domain is Azure CDN :-). @felixrieseberg, do we have an official azure storage account & CDN for this blog, if not I'm happy to stand one up on our team azure account.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 11 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: I did remove the link to my template on my personal repo with AWS specific variable names and just added `For the AWS scenario, the term 'local' referrs to the AWS VPC.`

I didn't mention the other two deployment paths because the cli works on any platform, with or without a UI. So its applicable for everyone. Nothing stopping folks from using other methods.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 12 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Nit is good. Fixed

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 13 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Great feedback. Done.

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 14 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Done

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 15 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: done

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 16 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: done

File: _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md (hunk 17 above)
Comment (sedouard) [inline _posts/2015-08-28-Connecting-an-AWS-VPC-to-Azure-VNet-via-Resource-Manager.md]: Doh. Copy + Paste error

**Comment by sedouard** [comment]
@felixrieseberg I think we're good here
